lib/matplotlib/rcsetup.py

```diff
@@ -1142,7 +1142,8 @@ def validate_cycler(s):
     'keymap.pan':          [['p'], validate_stringlist],
     'keymap.zoom':         [['o'], validate_stringlist],
     'keymap.save':         [['s', 'ctrl+s'], validate_stringlist],
-    'keymap.quit':         [['ctrl+w', 'cmd+w'], validate_stringlist],
+    'keymap.quit':         [['ctrl+w', 'cmd+w', 'q'], validate_stringlist],
+    'keymap.quit_all':     [['W', 'cmd+W', 'Q'], validate_stringlist],
```

Comment:
I don't think we need 3 keys for the same command. How about removing the "W" and "Q" and just having ```cmd+W```?